### DISCUSSION TITLE
When will new version of python kserve library get released?

### DISCUSSION THREAD

**Discussion by yangyang919** [kubeflow/kfserving#2129] [General]
/kind bug

**What steps did you take and what happened:**
There are some new changes are quite needed for us. We are wondering when a new version of kserve python library get released? Last release was in February

**Comment by yuzisun** [comment]
@yangyang919 which change you are looking for? we plan to release 0.9 mid May.

**Reply by yangyang919** [reply]
Hi @yuzisun , the change that upgrade for py3.8. https://github.com/kserve/kserve/commit/e7dfa0a33283135703cfe8747466199de7c62690
Only with this change, we can solve library conflict issue. Is there a chance new kserve version can be published as early as possible?

**Comment by yangyang919** [comment]
Hi @yuzisun , the change that upgrade for py3.8. https://github.com/kserve/kserve/commit/e7dfa0a33283135703cfe8747466199de7c62690
Only with this change, we can solve library conflict issue. Is there a chance new kserve version can be published as early as possible?

**Comment by yangyang919** [comment]
@yuzisun @Suresh-Nakkeran , I see that master branch of kserve already supports python3.8, I'm wondering when can we release a new version? It has been 4 months since last major version...We are really looking forward to version 0.9. We are blocked by old version over 1 month
